syntax = "proto3";

service MyApp {
  rpc io_bound (MyAppRequest) returns (MyAppResponse) {}
  rpc cpu_bound (MyAppRequest) returns (MyAppResponse) {}
}

message MyAppRequest {
  int32 delay = 1;
}

message MyAppResponse {
  string text = 1;
}
